observatory-platform/observatory/platform/cli/generate_command.py

```diff
@@ -221,6 +255,38 @@ def generate_workflow(self, project_path: str, package_name: str, workflow_type:
                 f.write(f'    {workflow_module} = "{workflow_module}"\n')
             print(f"- Updated the identifiers file: {identifiers_dst_file}")
 
+    def generate_local_config_interactive(self, config_path: str, *, workflows: Set[str], oapi: bool):
+        """Construct an Observatory local config file through user assisted configuration.
+
+        :param config_path: Configuration file path.
+        :param workflows: List of installer script installed workflows projects.
+        :param oapi: Whether installer script installed the Observatory API.
+        """
+
+        file_type = "Observatory Config"
+        click.echo(f"Generating {file_type}...")
+
+        config = InteractiveConfigBuilder.build(backend_type=BackendType.local, workflows=workflows, oapi=oapi)
+
+        config.save(config_path)
+        click.echo(f'{file_type} saved to: "{config_path}"')
+
+    def generate_terraform_config_interactive(self, config_path: str, *, workflows: Set[str], oapi: bool):
```

Comment:
Should workflows be a 'List[str]' instead? From the usage/tests it seems that it a list is passed on.